### PR TITLE
Enhanced instructions for enabling SASL for Kafka service

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

In response to the [Slack conversation](https://aiven-io.slack.com/archives/C02BW1F2WG7/p1691496136140139), the user encountered difficulty enabling SASL using the provided instructions. Incorporated additional steps based on the feedback received.

### CODE REVIEW

File: docs/products/kafka/howto/kafka-sasl-auth.rst [hunk 1]
```diff
@@ -1,18 +1,22 @@
 Use SASL Authentication with Apache Kafka®
```
Comment (wojcik-dorota) [inline docs/products/kafka/howto/kafka-sasl-auth.rst:1]: - Consider not capitalizing `authentication`.
- Consider adding `Aiven for` before `Apache Kafka`

Proposal: `Use SASL authentication with Aiven for Apache Kafka®`

File: docs/products/kafka/howto/kafka-sasl-auth.rst [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 
-Aiven offers a choice of :doc:`authentication methods for Apache Kafka® <../concepts/auth-types>`, including `SASL <https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer>`_ (Simple Authentication and Security Layer).
+Aiven offers a selection of :doc:`authentication methods for Apache Kafka® <../concepts/auth-types>`, including `SASL <https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer>`_ (Simple Authentication and Security Layer).
 
-1. Scroll down the **Service overview** page to the **Advanced configuration** section and select **Change**.
-
-2. Turn on the setting labelled ``kafka_authentication_methods.sasl``, and click **Save advanced configuration**.
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and choose your project. 
+2. From the list of services, choose the Aiven for Apache Kafka service for which you wish to enable SASL.
+3. On the **Service overview** page of the selected service, scroll down to the **Advanced configuration** section.
```
Comment (wojcik-dorota) [inline docs/products/kafka/howto/kafka-sasl-auth.rst:8]: `On the **Overview** page of ....`

File: docs/products/kafka/howto/kafka-sasl-auth.rst [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
+Aiven offers a selection of :doc:`authentication methods for Apache Kafka® <../concepts/auth-types>`, including `SASL <https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer>`_ (Simple Authentication and Security Layer).
 
-1. Scroll down the **Service overview** page to the **Advanced configuration** section and select **Change**.
-
-2. Turn on the setting labelled ``kafka_authentication_methods.sasl``, and click **Save advanced configuration**.
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and choose your project. 
+2. From the list of services, choose the Aiven for Apache Kafka service for which you wish to enable SASL.
+3. On the **Service overview** page of the selected service, scroll down to the **Advanced configuration** section. 
+4. Select **Change**.
+5. Enable the setting labeled ``kafka_authentication_methods.sasl``, and then select **Save advanced configuration**.
```
Comment (wojcik-dorota) [inline docs/products/kafka/howto/kafka-sasl-auth.rst:10]: `Enable the ``kafka_authentication_methods.sasl`` setting`

**Review by wojcik-dorota — APPROVED** [review]
A few minor editorial improvement ideas

File: docs/products/kafka/howto/kafka-sasl-auth.rst (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/howto/kafka-sasl-auth.rst]: Fixed

File: docs/products/kafka/howto/kafka-sasl-auth.rst (hunk 3 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/howto/kafka-sasl-auth.rst]: Done.

**Pull request merged**